spec.html

```diff
@@ -4490,51 +4490,59 @@ <h2>Syntax</h2>
-          <ul>
-            <li>
-              it is not `0`; or
-            </li>
-            <li>
-              there is a non-zero digit to its left and there is a non-zero digit, not in the |ExponentPart|, to its right.
-            </li>
-          </ul>
+          <p>The conversion of a String to a Number value is similar overall to the determination of the Number value for a numeric literal (see <emu-xref href="#sec-literals-numeric-literals"></emu-xref>), but some of the details are different, so the process for converting a String numeric literal to a value of Number type is given here. Additional definitions for MV are provided in <emu-xref href="#sec-static-semantics-mv"></emu-xref>.</p>
+          <emu-grammar>StringNumericLiteral ::: [empty]</emu-grammar>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace</emu-grammar>
+          <emu-alg>
+            1. Return *+0*<sub>𝔽</sub>.
+          </emu-alg>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace? StrNumericLiteral StrWhiteSpace?</emu-grammar>
+          <emu-alg>
+            1. Return 𝔽(MV of |StrNumericLiteral|).
+          </emu-alg>
+          <emu-grammar>StrDecimalLiteral ::: `-` StrUnsignedDecimalLiteral</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of |StrUnsignedDecimalLiteral|.
+            1. If _a_ is 0, return *-0*<sub>𝔽</sub>.
+            1. Return RoundStringMVResult(-1 &times; _a_).
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: `Infinity`</emu-grammar>
+          <emu-alg>
+            1. Return *+&infin;*<sub>𝔽</sub>.
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: DecimalDigits `.` DecimalDigits? ExponentPart?</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of the first |DecimalDigits|.
+            1. If the second |DecimalDigits| is present, let _b_ be MV of the second |DecimalDigits|. Otherwise, let _b_ be 0.
+            1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
+            1. Let _n_ be the number of code points in the second |DecimalDigits|.
+            1. Return RoundStringMVResult((_a_ + (_b_ &times; 10<sup>-_n_</sup>)) &times; 10<sup>_e_</sup>).
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: `.` DecimalDigits ExponentPart?</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of |DecimalDigits|.
+            1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
+            1. Let _n_ be the number of code points in |DecimalDigits|.
+            1. Return RoundStringMVResult(_a_ &times; 10<sup>_e_-_n_</sup>).
```

Comment:
```suggestion
            1. Return RoundStringMVResult(_a_ &times; 10<sup>_e_ - _n_</sup>).
```

Reply:
(ref issue #925)

Reply:
If the editors prefer the superscript without spaces, there are about 18 other superscripts *with* spaces that you might wish to convert.

Reply:
Thanks, I'll look into it.

Reply:
Let's keep spaces for this PR and decide the rendering afterwards.